src/neptune_tensorboard/integration/tensorflow_integration.py

```diff
@@ -13,255 +13,93 @@
-            if writer_name:
-                return writer_name + "_" + name
-        return name
-
-    def add_summary(self, writer, summary, global_step=None):
-
-        if isinstance(summary, bytes):
-            summ = summary_pb2.Summary()
-            summ.ParseFromString(summary)
-            summary = summ
-
-        x = self._calculate_x_value(global_step)
-        for value in summary.value:
-            try:
-                self.add_value(x, value, writer)
-            except NeptuneException:
-                pass
-
-    def add_value(self, x, value, writer):
-        field = value.WhichOneof("value")
-
-        if field == "simple_value":
-            self.send_numeric(
-                tag=self.get_channel_name(writer, value.tag), step=x, value=value.simple_value, wall_time=time.time()
-            )
-            return
-
-        if field == "image":
-            self.send_image(
-                tag=self.get_channel_name(writer, value.tag),
-                step=x,
-                encoded_image_string=value.image.encoded_image_string,
-                wall_time=time.time(),
-            )
-            return
-
-        if field == "tensor" and value.tensor.dtype == tf.string:
-            string_values = []
-            for _ in range(0, len(value.tensor.string_val)):
-                string_value = value.tensor.string_val.pop()
-                string_values.append(string_value.decode("utf-8"))
-
-            self.send_text(
-                tag=self.get_channel_name(writer, value.tag),
-                step=x,
-                text=", ".join(string_values),
-                wall_time=time.time(),
-            )
-            return
-
-    def send_numeric(self, tag, step, value, wall_time):
-        self._experiment_holder().send_metric(channel_name=tag, x=step, y=value, timestamp=wall_time)
-
-    def send_image(self, tag, step, encoded_image_string, wall_time):
-        image_desc = "({}. Step {})".format(tag, step)
-        self._experiment_holder().send_image(
-            channel_name=tag,
-            x=step,
-            y=Image.open(io.BytesIO(encoded_image_string)),
-            name=image_desc,
-            description=image_desc,
-            timestamp=wall_time,
-        )
-
-    def send_text(self, tag, step, text, wall_time):
-        self._experiment_holder().send_text(channel_name=tag, x=step, y=text, timestamp=wall_time)
-
-    @staticmethod
-    def get_writer_name(log_dir):
-        return os.path.basename(os.path.normpath(log_dir))
-
-    @staticmethod
-    def _calculate_x_value(global_step):
-        if global_step is not None:
-            return int(global_step)
-        else:
-            return time.time()
-
-
-def _integrate_with_tensorflow(experiment_getter, prefix=False):
-    tensorflow_integrator = TensorflowIntegrator(experiment_getter=experiment_getter, prefix=prefix)
-
+def _integrate_with_tensorflow(run, base_namespace):
     try:
         version = "<unknown>"
```

Comment:
This should be placed before `try ... exception` as it may not be visible in `except` block (not sure on what conditions it depends but PyCharm is even suggesting something similar)